tests/subsys/fs/fs_api/src/test_fs_dir_file.c

```diff
@@ -706,12 +1377,30 @@ void test_file_unlink(void)
 	ret = fs_unlink(NULL);
 	zassert_not_equal(ret, 0, "Delete a NULL file");
 
+	ret = fs_unlink("a");
+	zassert_not_equal(ret, 0, "Delete a file with too short name");
+
+	ret = fs_unlink("SDCARD:/test_file.txt");
+	zassert_not_equal(ret, 0, "Delete a file with wrong path");
```

Comment:
How "wrong" is the path? Why  nor write "Attempt deleting file with missing root / in path".